widget/toolbar.go

```diff
@@ -27,8 +27,14 @@ func (t *ToolbarAction) ToolbarObject() fyne.CanvasObject {
 	return button
 }
 
+// SetIcon updates the icon on a ToolbarItem
```

Comment:
As this is a new API we need to have a since line. Please add (including the line to separate):
```go
//
// Since: 2.2
```

The next feature release will probably be 2.2, so we pick this version in anticipation of the next larger release.